WordPress/Classes/Utility/Logging/WPCrashLogging.swift

```diff
@@ -72,6 +72,13 @@ extension WPCrashLogging {
         Client.shared?.appendStacktrace(to: event)
         Client.shared?.send(event: event)
     }
+
+    static func logMessage(_ message: String) {
```

Comment:
I have some doubts about this, and I wonder if this should go to Tracks instead, but then I'm excited to try Sentry's logging features 😁 

One thing I would suggest is that we don't send a bunch of different messages all related to previews failing, because I imagine it'll be hard to group them together. From what I understand, you can [send any extra properties you want](https://docs.sentry.io/clients/cocoa/advanced/#sending-events) with an event. We could have the same message for all the failed previews, then have a `reason` property in `extra`, that detailed whether permalink was nil, there were no credentials, or if there was any generic web view error.

Reply:
I agree with what's mentioned above.

On top of that, this function is called from `previewRequestFailed(message: String)`.  Isn't this an error message for that flow (instead of an informative one)?

I know there were some cases where this was being brought up in non-error scenarios and I was wondering if those are still valid.

Reply:
@diegoreymendez,
>Isn't this an error message for that flow (instead of an informative one)?
Not sure I understand what you mean.